content/2022-munich/register.md

```diff
@@ -6,6 +6,11 @@ title: Register
 
 Registration will open soon!
 
+## COVID / Health & Safety
+
+We are following both the [CNCF's guidance](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/health-and-safety/) and all German laws & regulations.
```

Comment:
This link is specific to KubeCon NA, which includes some sections about them having testing on-site on specific dates, etc. ("CNCF will provide free testing on-site from Sunday, October 23 – Friday, October 28"). It's not clear to me as the reader whether those things also generalize over to PromCon. E.g. do we also offer on-site testing, but at our dates? Could be good to clarify.